dev-libs/libappindicator/libappindicator-12.10.0-r200.ebuild

```diff
@@ -15,43 +17,56 @@ SRC_URI="http://launchpad.net/${PN}/${PV%.*}/${PV}/+download/${P}.tar.gz"
 LICENSE="LGPL-2.1 LGPL-3"
 SLOT="2"
 KEYWORDS="~amd64 ~x86"
-IUSE="+introspection"
+IUSE="introspection python"
 
 RDEPEND="
 	>=dev-libs/dbus-glib-0.98[${MULTILIB_USEDEP}]
 	>=dev-libs/glib-2.26:2[${MULTILIB_USEDEP}]
 	>=dev-libs/libdbusmenu-0.6.2[gtk,${MULTILIB_USEDEP}]
 	>=dev-libs/libindicator-12.10.0:0[${MULTILIB_USEDEP}]
-	x11-libs/gtk+:2[${MULTILIB_USEDEP}]
+	x11-libs/gtk+:2[${MULTILIB_USEDEP},introspection?]
 	introspection? ( >=dev-libs/gobject-introspection-1 )
+	python? ( ${PYTHON_DEPS} )
 "
 DEPEND="${RDEPEND}
 	virtual/pkgconfig[${MULTILIB_USEDEP}]
 	introspection? ( $(vala_depend) )
 "
 
-src_prepare() {
-	# Don't use -Werror
-	sed -i -e 's/ -Werror//' {src,tests}/Makefile.{am,in} || die
+multilib_src_prepare() {
+	if ! multilib_is_native_abi || ! use python; then
+		cd "${BUILD_DIR}"
 
-	epatch "${FILESDIR}"/${P}-disable-python.patch
-	eautoreconf
+		epatch "${FILESDIR}"/${P}-disable-python.patch
```

Comment:
No love for conditional patching. Please prepare a proper patch adding some kind of `--enable-python`.